src/sage/combinat/root_system/non_symmetric_macdonald_polynomials.py

```diff
@@ -421,99 +433,108 @@ class NonSymmetricMacdonaldPolynomials(CherednikOperatorsEigenvectors):
         sage: KL0 = KL.classical()
         sage: P = K['x0,x1']
         sage: def EE(weight): return E[L0(weight)].expand(P.gens())
+
+        sage: # needs sage.combinat
         sage: import sage.combinat.sf.ns_macdonald as NS
         sage: EE([0,0])
         1
-        sage: NS.E([0,0])
+        sage: NS.E([0,0])                                                               # needs sage.groups
```

Comment:
could use block scope for `sage.groups` too

Reply:
Here doctests that need sage.groups are alternating with doctests that don't; this information would be lost with a block tag